app/css/style.css

```diff
@@ -7,6 +7,20 @@ html, body, iframe {
 
 md-card[md-ink-ripple] {
   position: relative;
+  width:100%;
+  /*overflow-y: auto;*/
+}
+
+p {
```

Comment:
Maybe it is not a good idea to change every `<p>`
Could you make this specific to some class?